core/src/main/java/com/netease/arctic/scan/KeyedTableScan.java

```diff
@@ -45,4 +45,13 @@ public interface KeyedTableScan {
    * possible during execution.
    */
   KeyedTableScan enableSplitTaskByDeleteRatio(double splitTaskByDeleteRatio);
+
+  /**
+   * Config this scan using the given reference.
+   *
+   * @param ref reference
+   * @return scan based on the given reference.
+   * @throws IllegalArgumentException if a reference with the given name could not be found
+   */
+  KeyedTableScan useRef(String ref);
```

Comment:
Does this interface method need to be introduced in this PR?

Reply:
I removed it.